src/stylesheets/_variables.scss

```diff
@@ -0,0 +1,13 @@
+// Colors
+$white: #fff !default;
+$black: #000 !default;
+$primary-color: $black !default;
+$secondary-color: $white !default;
+$black-button-text: #15151a !default;
+
+// Font Size
+$semi-font-size: rem(17px);
```

Comment:
@rafayet-monon For `font-size`, I think we can group them like this.
```scss
// Typography

$font-sizes: (
  default: 1rem,  // 16px
  xxs: 0.6875rem, // 11px
  xs: 0.75rem,    // 12px
  sm: 0.875rem,   // 14px
  md: 0.9375rem,  // 15px
  lg: 1.0625rem,  // 17px
  xl: 1.25rem,    // 20px
  xxl: 1.5rem,    // 24px
  xxxl: 1.85rem,  // 29.6px
  xxxxl: 2rem     // 32px
);
```

And this is how we use it.
```scss
label {
  font-size: map-get($font-sizes, 'xxl');
}
```